src/freenet/clients/http/QueueToadlet.java

```diff
@@ -134,7 +134,7 @@ void setFIW(FileInsertWizardToadlet fiw) {
 	private boolean isReversed = false;
 	private final boolean uploads;
 
-    private static final String FETCH_KEY_LIST_LOCATION = "listFetchKeys.txt";
+    private static final String KEY_LIST_LOCATION = "listKeys.txt";
```

Comment:
Waiting on responses to posts on FMS/Sone/Frost if the old URI is being used by anyone that insists on it staying the same.  if someone complains in the next few days, I'll add an OLD_KEY_LIST_LOCATION = "listFetchKeys.txt" here and alter L#1076 accordingly.